python_training/test_me.py

```diff
@@ -0,0 +1,48 @@
+import pytest
+from .pycalculator import Calculator
+
+
+@pytest.fixture()
+def calculator():
+    return Calculator()
+
+
+def test_add_a_to_b(calculator):
```

Comment:
That's a classic case for `@parametrize`, since we'll want to check stuff like negative results, addition with zero, etc..
In fact it can be used in most of the tests here, for multiple cases.